### PR TITLE
Feature/e2e tests poc

### PR DESCRIPTION
**Pull request by warm-coolguy**
## Summary

Added playwright as E2E framework with rather trivial example test. With this, we can test some "big picture" effects.

## Instructions for local reproduction and review

`npm i`, `npx playwright install` to install browsers for your system once (from what I understand, this has to be done once per executing machine), `npm run test:e2e`.

## TODO

- [x] Add test for draw tool (actually drawing in map to have an example for canvas interactions)

### CODE REVIEW

**Comment by dopenguin** [comment]
The pipeline for the E2E tests are currently failing. 🏓 me, once the lights are green!

**Comment by warm-coolguy** [comment]
> The pipeline for the E2E tests are currently failing. 🏓 me, once the lights are green!

🏓 @dopenguin 

Also added another test to display canvas interaction possibilities.

**Review by dopenguin — CHANGES_REQUESTED** [review]
🏓 @warm-coolguy

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,8 @@ logs
 /.idea/
 /.nx/
 **/.parcel-cache
+node_modules/
```
[the remark this answers is not in the document]
Comment (warm-coolguy) [reply .gitignore]: Yep, removed it.

File: playwright.config.ts
[the remark this answers is not in the document]
Comment (warm-coolguy) [reply playwright.config.ts:1]: We deleted some lines on the telephonator. ☎️

**Comment by warm-coolguy** [comment]
> 🏓 @warm-coolguy

🏓 @dopenguin

**Review by dopenguin — APPROVED** [review]
🚀 @warm-coolguy

**Pull request merged**